code/hud/hudconfig.cpp

```diff
@@ -180,6 +180,10 @@ int HUD_default_popup_mask2 =
 	0											// kills gauge
 };
 
+// Maybe let mods define these eventually? I dunno how much FOTG might care about HUD Config showing mod appropriate
+// wing names?
+char HC_wingam_gauge_status_names[MAX_SQUADRON_WINGS][32] = {"Alpha", "Beta", "Gamma", "Delta", "Epsilon"};
```

Comment:
Ah good question... Yeah it would be very useful if this could be a string list in the HUD table since we do indeed use different designations, but I can add that as a PR later

Reply:
I was also wondering if doing an XSTR might be good since then mods could easily change it(though it would just need a <=32 length check of course)...

Reply:
Doing "Wing 1, Wing 2, Wing 3" would work except that, depending on the mod's settings it may draw as WIN, WIN, WIN instead. Table or XSTR is better for customization. Probably table so that mods can be explicit about what wing names show up here and can then also take advantage of built-in XSTR support that exists in `stuff_string()`.

Reply:
Ah that's right, and table sounds good! Overall, everything else looks good, so we can coordinate on a table entry on a separate PR if that sounds good to you?

Reply:
I guess we could add a TODO comment about the plan to make this table specific, and then just remove the FOTG line?

Reply:
I have added the table parser. I put it at the top so that it will cause a code conflict with #6581 and I can remember to integrate that into the `#HUD Config Settings` section parser that PR creates.

Reply:
Oh awesome, thanks!